consensus/src/pipeline/virtual_processor/utxo_validation.rs

```diff
@@ -177,7 +179,7 @@ impl VirtualStateProcessor {
 
         // Verify all transactions are valid in context
         let current_utxo_view = selected_parent_utxo_view.compose(&ctx.mergeset_diff);
-        let validated_transactions =
+        let (validated_transactions, _) =
```

Comment:
This is a frequent call (called per chain block), which does not use the combined muhash (thus adding significant cost not present beforehand). 
I think we should keep `validate_transactions_in_parallel` like it was, and simply apply a new parallel operation for the muhash reduce to be applied only within `calculate_utxo_state`

Reply:
or create two different `validate_transactions_in_parallel` impls